src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -432,13 +416,10 @@
                 <models:Entry />
             </ContentDialog.DataContext>
             <ScrollViewer
-                HorizontalScrollBarVisibility="Auto"
-                HorizontalScrollMode="Auto">
+                Padding="0,0,16,0">
                 <StackPanel
-                    Width="480"
```

Comment:
Yeah, `480` is a bit too much.. However, the current dialog feels cramped and is not great to read when having a comment that is longer than a few words:

<img width="257" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/373c5986-6e1c-430d-933a-e26f5d5b26fe">

Also the left/right margin of the fields do not seem to be equal?

Reply:
Before, the (min)width of the content was equal to the width of the parent window, forcing horizontal scrolling. Quickest solution was to remove the restriction. How long can comments get? To me, it doesn't look so bad. But 320 is also good.

And yes, I added a right-padding to make sure the vertical scrollbar is always pretty. Was that bad?
Note: the ContentDialog has a padding of itself, so forcing the scrollbar/scrollviewer to the edge doesn't feel fair to me.

![image](https://github.com/microsoft/PowerToys/assets/65828559/23bdb225-7786-4093-917b-e70cf6717786)
![image](https://github.com/microsoft/PowerToys/assets/65828559/fa9749b1-a28c-411f-924b-24da59b7da06)